### PR TITLE
Updating documentation to MQTT (See Pull 7455)

### PR DESCRIPTION
**Pull request by bberg115**
As a request of user fabaff, moving change notes to Add-on Section instead of MQTT generic

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_addons/mosquitto.markdown [hunk 1]
```diff
@@ -55,6 +55,22 @@ This add-on is attached to the Home Assistant user system, so mqtt clients can m
 
 To use the Mosquitto as [broker](/docs/mqtt/broker/#run-your-own), go to the integration page and install the configuration with one click. If you have old MQTT settings available, remove this old integration and restart Home Assistant to see the new one.
 
+
+  #### Using Mosquitto with Hass.io
```
Comment (klaasnicolaas) [inline source/_addons/mosquitto.markdown]:
```suggestion
  #### {% linkable_title Using Mosquitto with Hass.io %}
```

File: source/_addons/mosquitto.markdown [hunk 2]
```diff
@@ -55,6 +55,22 @@ This add-on is attached to the Home Assistant user system, so mqtt clients can m
...
+  Install the [Mosquitto add-on](https://www.home-assistant.io/addons/mosquitto/) with the default configuration via 'Hass.io > ADD-ON STORE'. (Don't forget to start the add-on & verify that 'Start on boot' is enabled.)
+   Next, create a new user for MQTT via the `Configuration > Users (manage users)`. Restart your HA. (Note: This name cannot be "homeassistant" or "addon")
+   Once back on-line, return to `Configuration > Integrations` and select configure next to `MQTT`.
+   ##### Example of MQTT Integration
```
Comment (klaasnicolaas) [inline source/_addons/mosquitto.markdown]: Perhaps put this as `# Example of MQTT Integration` under line 64

File: source/_addons/mosquitto.markdown (hunk 1 above)
Comment (frenck) [reply source/_addons/mosquitto.markdown]: Resolved.

File: source/_addons/mosquitto.markdown (hunk 2 above)
Comment (frenck) [reply source/_addons/mosquitto.markdown]: Resolved.

**Review by frenck — APPROVED** [review]

**Pull request merged**